src/pages/costModels/components/rateTable.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { Rate } from '../../../api/rates';
```

Comment:
If you could, please replace relative paths with absolute paths

Reply:
Thanks. I fixed that.
In addition, I checked that `I18n` uses the correct locale. Although I had to also set `fallback` to `fr` in the `I18nProvider.tsx`.